### ISSUE TITLE
Runner.run parameter not available

### ISSUE THREAD

**Issue by Sabou44** [DevExpress/testcafe#6315]
Hi,

In the documentation it's said that it's possible to use options to configure the runner. I would like to use the disableMultipleWindows option
```
runner.run({ disableMultipleWindows: true });
```
But I get the message :
```
Argument of type '{ disableMultipleWindows: boolean; }' 
is not assignable to parameter of type 'Partial<RunOptions>'
```
Is this a bug or is this option deliberately not activated yet?

I am using the latest stable release v1.14.2

**Comment by viktoria2506** [comment]
Thank you for submitting this issue. I'm happy to help you diagnose this behavior, but I couldn't reproduce it. Please describe your environment in greater detail and share a sample project and tests. Then, we will be able to determine the cause of the issue. ​I look forward to your response.

**Comment by Sabou44** [comment]
Hi,

here is my simple example:

[disableMultipleWindows.zip](https://github.com/DevExpress/testcafe/files/6676625/disableMultipleWindows.zip)

As I already mentioned in my question I use v1.14.2

**Comment by Dmitry-Ostashev** [comment]
Thank you for your example. I reproduced the issue. The disableMultipleWindow option is missing in type definitions. As a workaround, you can specify it in the following way:

```
    const options: any = { disableMultipleWindows: true };

    return runner.run(options);
```

**Comment by Sabou44** [comment]
Hi @Dmitry-Ostashev,

Thanks for the quick answer. It works well !
I had seen that the option was missing in the type definitions but I didn't know if it was intentional or not. I'm glad I could help you to detect this oversight :)

**Comment by miherlosev** [comment]
Fixed in `testcafe@1.17.2-rc.2`.
